aspnet/security/cross-site-scripting.rst

```diff
@@ -1,8 +1,256 @@
-.. include:: /../common/stub-topic.txt
+.. _security-cross-site-scripting:
 
-|stub-icon| Preventing Cross-Site Scripting
-===========================================
+Preventing Cross-Site Scripting
+===============================
 
-.. include:: /../common/stub-notice.txt
+Cross-Site Scripting (XSS) is a security vulnerability which enables an attacker to place client side scripts (usually JavaScript) into web pages.
+When other users load affected pages the attackers scripts will run, enabling the attacher to steal cookies and session tokens, change the contents
+of the web page via DOM manipulation or redirect the browser to another page. XSS vulnerabilities generally occur when an application takes user 
+input and outputs it in a page without validating or encoding or escaping it.
```

Comment:
Probably should stick with your double "or" but
 _without validating, encoding or escaping it._
reads better at the loss of the explicit or.